### PR TITLE
ERMS-2654, ERMS-2679

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy [hunk 1]
```diff
@@ -5437,7 +5436,7 @@ class SubscriptionController
                                     owner: dctx.owner.owner
                             ).save()
 
-                            String fPath = ConfigUtils.getDocumentStorageLocation() ?: '/tmp/laser'
+                            String fPath = grailsApplication.config.documentStorageLocation ?: '/tmp/laser'
```
Comment (klodav) [inline app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:5439]: regression

Comment (Moetez16) [reply app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:5439]: Upps

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy [hunk 2]
```diff
@@ -34,7 +34,6 @@ import org.codehaus.groovy.grails.plugins.orm.auditable.AuditLogEvent
 import org.codehaus.groovy.runtime.InvokerHelper
 import org.springframework.context.i18n.LocaleContextHolder
 import org.springframework.web.multipart.commons.CommonsMultipartFile
-import de.laser.helper.ConfigUtils
```
Comment (croqueGrec09) [inline app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:37]: Why?

File: app/grails-app/views/subscription/_nav.gsp [hunk 3]
```diff
@@ -28,6 +28,10 @@
         </sec:ifAnyGranted>
     </g:if>
 
+    <g:if test="${((contextService.org?.getCustomerType() in ['ORG_CONSORTIUM']) && subscriptionInstance.instanceOf)}">
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/_nav.gsp:31]: often said ... please be careful with copy-paste! Then, contextService.org --> (result.)institution!

File: app/grails-app/views/subscription/_nav.gsp [hunk 4]
```diff
@@ -28,6 +28,10 @@
...
+    <semui:securedSubNavItem orgPerm="ORG_CONSORTIUM" controller="subscription" action="surveys" params="${[id:params.id]}" message="subscription.details.surveys.label" />
+    </g:if>
+
     <g:if test="${((contextService.org?.getCustomerType() in ['ORG_INST', 'ORG_BASIC_MEMBER']) || params.orgBasicMemberView)&& subscriptionInstance?.type == de.laser.helper.RDStore.SUBSCRIPTION_TYPE_CONSORTIAL}">
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/_nav.gsp:35]: see above - contextService.org is always set!

**Pull request merged**